content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -3,135 +3,211 @@ title: Instrumentation
 weight: 3
 ---
 
-This guide will cover creating and annotating spans, creating and annotating metrics, how to pass context, and a guide to automatic instrumentation for JavaScript. This simple example works in the browser as well as with Node.js
+Manual instrumentation is the process of adding observability code to your
+application.
 
-## Example Application
+## Initialize Tracing
 
-In the following this guide will use the following sample app:
+To start [tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry),
+you'll need to have an initialized
+[`TracerProvider`](/docs/concepts/signals/traces/#tracer-provider) that will let
+you create a [`Tracer`](/docs/concepts/signals/traces/#tracer).
 
-```javascript
-'use strict';
+If a `TracerProvider` is not created, the OpenTelemetry APIs for tracing will
+use a no-op implementation and fail to generate data.
 
-for (let i = 0; i < 10; i += 1) {
-  doWork();
-}
+### Node.js
+
+To initialize tracing with the Node.js SDK, first ensure you have the SDK
+package and OpenTelemetry API installed:
+
+```shell
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/sdk-trace-node
+```
+
+Next, create a separate `tracing.js` file that has all the SDK initialization
```

Comment:
I am just wondering why the code in `tracing.js` is separated out from the application itself, what's the goal here? Make it easier to swap out the SDK with a vendor specific one? 

If so, the sample app still needs to load the API to work without the `-r tracing.js` I assume? 

This will also be affected from the issue https://github.com/open-telemetry/opentelemetry-js/issues/2912. 

The sample app is also not lined out in the in the document anymore, which makes it hard for the enduser to follow those steps by simply doing a "copy&paste".

@dyladan you know this probably best, any thoughts?

Reply:
I've seen this as a pretty common thing I've seen, to isolate all initialization stuff (and instrumentation registration) into a separate file and then just use the API in the other files in an app where manual instrumentation is done. I'm fine with changing this; whatever comports most with how you'd initialize opentelemetry in a real-world app.

As for a sample, I could also add it back in, but unlike a getting-started flow this doc tends to be more of a "I know I want to do a thing, let me quickly find that subtopic and let me copy-paste some code into my own app and then tweak it so it compiles/runs as-needed" sort of thing. I don't think having a single sample that you run through from top to bottom is as important here.

Reply:
One very common pitfall regarding instrumentations is that it's needed to first create/register the OTel instrumentation and then require the corresponding module.

By using a dedicated file which is preloaded (or required at the beginning of of the main script) this works more or less automatic.
If you put the OTel init code into your main script one ends up with a Otel section (require + code) and then again 
some require lines and code. Most people don't like require lines after code sections so they tend to resort it - and break their OTel setup.

Reply:
> Maybe it's worthwhile do have a section explaining exactly that, i.e. using exactly your words why you might like to fork out otel setup into it's own file and then either require it via -r tracing.js or a `require("tracing.js") statement?

I've made a similar comment on the other PR but i understand that we want to avoid to overwhelm readers with too much "gotcha" to know. However it might be worth to add a Q&A section later on to document these